app/routes/webhooks.py

```
@@ -0,0 +1,85 @@
+        user_data = WebhookUserData(**(data))
+
+        # Check if user already exists
+        existing_user = await User.filter(user_id=user_data.id).exists()
+        logger.info(f"User exists check for {user_data.id}: {existing_user}")
+
+        if existing_user:
+            logger.info(f"User {user_data.id} already exists, skipping creation")
+            return
+
+        # Create user using the validated Pydantic model data
+        user = await User.create(
```

Ruff (0.11.9)

Local variable `user` is assigned to but never used

Remove assignment to unused variable `user`

(F841)